addons/mail/static/src/components/emoji_picker_header_action_view/emoji_picker_header_action_view.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.EmojiPickerHeaderActionView" owl="1">
+        <div class="o_EmojiPickerHeaderActionView d-flex" t-attf-class="{{ className }}" t-ref="root">
```

Comment:
The action icon in knowledge should be properly centered with the action itself. I think I put `d-flex` to make sure it's nicely centered, otherwise it's a bit offset.

IMO everything should be `d-flex` by default. The only reason we don't do flex enough is because it's more effort to add `d-flex` rather than just don't but anything. In the meantime it's okay to just put `d-flex` so there's no unsurprising broken layout with implicit `d-block` or `d-inline(-block)`, until we definitely decide to make `d-flex` the proper default and we don't need to explicitly tell `d-flex` in template.